lib/libmbedtls/sub.mk

```diff
@@ -3,6 +3,7 @@ global-incdirs-y += mbedtls/include
 
 # OBJS_CRYPTO from make file
 SRCS_CRYPTO :=
+ifneq ($(CFG_CRYPTOLIB_NAME_mbedtls),y) # not core
```

Comment:
Does this really work as intended?
What happens in the user TA build if `CFG_CRYPTOLIB_NAME_mbedtls=y`?

Reply:
I need to double check but given that we have below:
```make
ifeq ($(CFG_CRYPTOLIB_NAME_mbedtls),y)
subdirs-$(sm-core) += core
endif
```
...it must be OK.

Reply:
Oh no scratch that I missed the `$(sm-core)` :-/ sorry I'll fix